dom.bs

```diff
@@ -5762,13 +5785,19 @@ It is initially set to false.</p>
 <p><a for=/>Shadow roots</a> have an associated
 <dfn export for=ShadowRoot>available to element internals</dfn>. It is initially set to false.</p>
 
+<p><a for=/>Shadow roots</a> have an associated <dfn export for=ShadowRoot>is declarative shadow
```

Comment:
No wrapping inside a phrasing-level element. So wrap before `<dfn` I suspect.

Reply:
Also, I think we can just call this "declarative" or some such as it's directly tied to a shadow root so no need to go for such a long name.